app/models/abstract_feature.rb

```diff
@@ -134,37 +173,64 @@ def self.geojson(lowres: false, precision: 6, properties: true, srid: 4326, cent
       column = 'geog'
     end
 
-    properties_sql = <<~SQL if properties
-      , 'properties', hstore_to_json(metadata)
+    properties_sql = []
+
+    if include_record_identifiers
+      properties_sql << "hstore(ARRAY['feature_name', name::varchar, 'feature_id', id::varchar, 'spatial_model_type', spatial_model_type::varchar, 'spatial_model_id', spatial_model_id::varchar])"
+    end
+
+    if properties
```

Comment:
If properties is an explicit hash, do we still want to do this part?

Reply:
Probably not. In fact, we can probably take out 'features_area' as it is too tightly coupled to CE, and is actually no longer even used there.